tools/run_tests/python_utils/filter_pull_request_tests.py

```diff
@@ -129,7 +129,7 @@ def _get_changed_files(base_branch):
     base_commit = check_output(["git", "merge-base", base_branch,
                                 "HEAD"]).rstrip()
     return check_output(["git", "diff", base_commit, "--name-only",
-                         "HEAD"]).splitlines()
+                         "HEAD"]).decode("UTF-8").splitlines()
```

Comment:
nit: Maybe we can update the `check_output` function itself? Then we don't need `decode()` in other places.

Reply:
We can't, this is python's subprocess.check_output. For whatever reason, it was imported as `from subprocess import check_output`. I've changed the import to make that clear.